test/Test/Database/LSMTree/ModelIO/Normal.hs

```diff
@@ -201,7 +203,50 @@ prop_insertLookupRange h ups k v r = ioProperty $ do
 -- implement classic QC tests for split-value BLOB retrieval
```

Comment:
How about adding props that are analogous to `prop_lookupDelete` and co?

Reply:
`prop_lookupDelete` is already blob-aware.

```haskell
-- | You cannot lookup what you have just deleted
prop_lookupDelete ::
     IsTableHandle h
  => Proxy h -> [(Key, Update Value Blob)]
  -> Key -> Property
prop_lookupDelete h ups k = ioProperty $ do
    (_, hdl) <- makeNewTable h ups
    deletes hdl [k] -- this key may have a blob attached, may not. Depends on the table generated above.
    res <- lookupsWithBlobs hdl [k]
    return $ res === [NotFound k]
```